amrit-docker-setup/docker-compose.yml

```
@@ -0,0 +1,457 @@
+services:
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)